scripts/process_notebooks.py

```diff
@@ -277,12 +277,12 @@ def extract_solutions(nb, nb_dir, nb_name):
 
                     tag_args = " ".join([
                         "alt='Solution hint'",
-                        "align='left'",
+                        "align='center'",
```

Comment:
Adding a space make sense by why change the alignment?

Reply:
It seemed to need both to fix the layout issue (no idea why...)

Reply:
But it's going to change how the notebooks look on colab too, right? I think having the solution image centered and the actual image left-aligned will be annoying (I think we tried this last year and it looked bad).

Reply:
Yeah it would... Let me see if I could just change for the jupyterbook

Reply:
oh super easy to do by parsing the html just for jupyterbook I think. Will do that instead